src/routes/blog/spyder-6-remote-development/+page.md

```diff
@@ -0,0 +1,94 @@
+
+The foundations of Spyder's new remote development platform are available now in Spyder 6.0, with more features to come in 6.1 and beyond!
+The goal is to make it easy for you to host their files, data and computations on remote desktops, clusters, servers and the cloud while being able to use Spyder just like you normally would locally.
+
+Alongside built-in backend and client-server components, Spyder 6.0 features a new remote connection manager that allows you to connect to any machine with a SSH server, no prior installation or configuration needed, and automatically set up and use a remote Python environment  on the host.
+It also features graphical status checking and reporting, host configuration management, and more.
+In addition, it brings a new `Consoles` submenu allowing you to connect to any remote environment with one click, launch a console and run your code.
+You can also stop code execution as well as restart remove kernels from right within Spyder.
+
+<video controls>
+  <source src="remote-console-demo.mp4" type="video/mp4">
+  <track kind="captions">
+  Sorry, your browser doesn’t support HTML5 video. Download the <a href="screencast.mp4">MP4 file</a>
+</video>
+
+We're excited to have our newest core developer, Hendrik Louzada, and our team share with you their insights and perspectives on Spyder's new remote development platform introduced in Spyder 6!
+Join us as Hendrik shares how he got started with Spyder and the project, how the new achetecture is implemented under the hood, what challenges we faced and what he learned from them, and what's next for Spyder 6.1.0!
+
+
+## How'd you find out about Spyder and why'd you join the project?
+
+Hendrick first started to work on Spyder around three years ago, developing a series of plugins to control remote magnetic resonance equipment (such as MRI machines) directly in the IDE.
+As part of that project, he made multiple upstream contributions to Spyder itself to improve the Language Server Protocol support (LSP, the architecture that powers Spyder's code completion, introspection, analysis and formatting).
+
+In 2023, Spyder received a [Chan Zuckerberg Initiative](https://chanzuckerberg.com/) (CZI) [EOSS 5 grant](https://chanzuckerberg.com/eoss/proposals/enhancing-spyder-ide-remote-support-for-scientific-research-in-python/) to implement a new remote development platform and features in Spyder to allow users to develop and run code in remote desktops, servers, clusters and cloud machines.
+After a six-month search by Spyder lead maintainer Carlos Cordoba to find the right candidate to engineer the backend and network architecture for this project, he found Hendrik, who joined the core development team near the end of 2023.
+
+
+![New remote connection manager dialog in Spyder listing the configurable settings for a new remote host](remote-connection-manager-new.png)
```

Comment:
The image is already directly above the paragraph (and section) it most directly relates to, so there's not really an issue—in many news articles, images often if not typically _precede_ their relevant headings, as is the case here.

By contrast, there are several problems with moving it:

   * The primary purpose of the images in a personal interview post as in most blog posts/articles of this nature, (as opposed to a explanatory, how-to, tutorial, etc. post) are to break up a wall of text, provide provide visual interest and offer a background illustration of the topic, rather than to provide critical, immediately relevant descriptive information.
   * Moving that below would break the consistent pattern of an image after each (non-ending) section and cram two images only a short paragraph apart while leaving a larger wall of text between
   * Furthermore, it leads the reader to question why that specific paragraph/section gets special explanatory treatment and not the rest, and disrupts the feel of an interview (focused on a person) as opposed to a motivated feature how-to (focused just on the feature)
   * And it leads the reader to expect that other images will be directly below the paragraph it most relates to, which is not the case

Reply:
Agreed with @ccordoba12 to add line breaks and a caption to images instead